packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/README.md

```diff
@@ -66,6 +66,14 @@ Tests are written with [`vitest`](https://vitest.dev/). Snapshot tests are using
 [`@testing-library/react`](https://testing-library.com/docs/react-testing-library/intro/).
 Some example tests are provided.
 
+## Decoupled Kit Health Check
+
+The `@pantheon-systems/decoupled-kit-health-check` will run before your `build`
+and ensure the backend is setup for success. Helpful instructions will guide you
+in case anything in the health check fails. To disable the health check, see
+[Disabling the Decoupled Kit Health Check](https://live-decoupled-kit-docs.appa.pantheon.site/docs/frontend-starters/gatsby/gatsby-wordpress/troubleshooting#disabling-the-decoupled-kit-health-check).
```

Comment:
@CobyPear Any reason we're not using the https://decoupledkit.pantheon.io/ vanity domain for this link?

Reply:
Nope :) will fix.